app/modules/entities/components/layouts/other_editions.svelte

```diff
@@ -52,21 +46,22 @@
       <Link
         url={`/entity/${workUri}`}
         classNames="work-button"
-        html={I18n('see_all_work_editions', { label: currentEdition.label })}
+        html={I18n('see_all_work_editions', { label: workLabel })}
         grey={true}
         tinyButton={true}
       />
-	  </div>
+    </div>
 	{/if}
 {/await}
 
 <style lang="scss">
   @import '#general/scss/utils';
-  .other-editions{
+  .other-edition{
```

Comment:
The plural was fine no? It's still other editions from one of the edition works
```suggestion
  .other-editions{
```

Reply:
done in 083836a